core/node/eth_watch/README.md

```diff
@@ -2,3 +2,16 @@
 
 This crate contains implementation of zkSync Era Eth Watcher component, responsible for fetching the changes from the
 corresponding L1 contract.
```

Comment:
nit:
I know you did not touch this, but  if you feel comfortable, you may change this so it looks grammarly better
```
This crate contains an implementation of the zkSync Era Eth Watcher component, which fetches the changes from the corresponding L1 contract.
```